ui/src/pages/ShoeMap.tsx

```diff
@@ -1,11 +1,35 @@
 import * as React from "react";
+import styled from "styled-components";
+import { Map, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
```

Comment:
We should move all the css dist files to a common file and import them into `index.tsx`.

Reply:
This file is imported from the leaflet package. Could you elaborate?

Reply:
You know what, after looking at this more, I think it's fine as is, ignore my point lol.